request-manager/status/manager.go

```diff
@@ -50,10 +50,43 @@ func NewManager(dbc myconn.Connector, jrClient jr.Client) Manager {
 func (m *manager) Running(f Filter) (proto.RunningStatus, error) {
 	status := proto.RunningStatus{}
 
-	running, err := m.jrc.SysStatRunning()
+	ctx := context.TODO()
+	conn, err := m.dbc.Open(ctx)
+	if err != nil {
+		return status, err
+	}
+	defer m.dbc.Close(conn) // don't leak conn
+
+	// Make a list of the URLs of all JR hosts currently running any requests.
+	q := "SELECT jr_url FROM requests WHERE state = ? AND jr_url IS NOT NULL"
```

Comment:
Could use `SELECT DISTINCT jr_url FROM...` to avoid map, i.e. make MySQL de-dupe the list. In this simple case, and having an index on `state` col already, it's probably same  query perf for MySQL. Don't have to change it, just noting.